<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>comics_widget</class>
 <widget class="QDialog" name="comics_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_mon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>40</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Mon</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tue">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>8</y>
     <width>40</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Tue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_wed">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>8</y>
     <width>40</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Wed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_thu">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>8</y>
     <width>40</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Thu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_fri">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>8</y>
     <width>40</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Fri</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sun">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>8</y>
     <width>40</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Sun</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sat">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>8</y>
     <width>40</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Sat</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>160</width>
     <height>208</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>160</width>
     <height>208</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_box_status">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>236</y>
     <width>302</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>301</width>
     <height>169</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollArea_box">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>299</width>
      <height>167</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>301</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="layout_comics"/>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="btn_layout"/>
  </widget>
  <zorder>text_box</zorder>
  <zorder>btn_mon</zorder>
  <zorder>btn_tue</zorder>
  <zorder>btn_wed</zorder>
  <zorder>btn_thu</zorder>
  <zorder>btn_fri</zorder>
  <zorder>btn_sun</zorder>
  <zorder>btn_sat</zorder>
  <zorder>image_label</zorder>
  <zorder>text_box_status</zorder>
  <zorder>scrollArea</zorder>
  <zorder>verticalLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
